korok/src/doc_parsing/parsing_utils.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+# todo general question: do we interpret a different capitalization of the same word as the same word? Need to address
+
+# todo had to remove pyproject.toml to run, need to debug
+
+
+def parse_by_word_position(text: str) -> List[List[str]]:
+    """
+    Split a text into words in lines
+    :param text: Input text, may contain multiple lines
+    :return: A list of lines, each line is a list of separate words, no white spaces
+    """
+    lines = text.split('\n')
```

Comment:
there is a method splitlines or something

Reply:
Zodek
https://github.com/johnny423/korok/pull/2